app/assets/stylesheets/components/common/_footer_secondary.scss

```diff
@@ -104,14 +104,20 @@
   }
 }
 
-.footer-secondary__sfgb-text {
+.footer-secondary__mps-text {
   display: inline-block;
+}
+.footer-secondary__mps-text--en {
+  width: 98px;
   & > * {
     display: block;
```

Comment:
:question:  this `display: block` declaration being only present in the `EN` version is something we need to have or we forgot to copy it? (or may not be necessary at all).

Reply:
Nope, that's because the text in English and Welsh is different lengths and needs to break differently. I'm going to chat with @davidtrussler about this today and get his opinion, this may well change, will update here when we come to a conclusion

Reply:
@scruti So I've chatted to @davidtrussler about this, and we think the best way to do this for the user is to set the line breaks in the translation files. I've committed with those changes, which has removed this code entirely